lib/wholebodycontrollib/statemachine.py

```diff
@@ -77,36 +77,37 @@ def get_state(self, use_parametrized=False, ref=0, pos=0, J=0, period=0.001, tra
 
         com_position_initial = self.configurations[self.current_state - 1].com_position
         com_position_final = self.configurations[self.current_state].com_position
-
+        
         if self.current_state==1 and use_parametrized:
             self.pos_offset = pos - ref
+            
 
         if self.current_state==2 and use_parametrized:
 
-            # ref = ref + (self.pos_offset)
-            ref = ref + self.compute_offset(ref)
-            # print('ref ' + str(ref))
-            # print('pos ',str(pos))
-
-            if ref < 0.8:
-                if ref > pos:
-                    tracking_gain  = tracking_gain
-                if ref < pos and ref>0.7:
-                    tracking_gain =  tracking_gain * ((ref - 0.7)/0.1)
-                if ref<0.7:
-                    tracking_gain = 0
-
-                if self.happy:
-                    self.rpc_command.addString('shy')
-                    self.rpc_client.write(self.rpc_command, self.rpc_response)
-                    self.rpc_command.pop()
-                    self.happy = False
-            else:
-                if not self.happy:
-                    self.rpc_command.addString('happy')
-                    self.rpc_client.write(self.rpc_command, self.rpc_response)
-                    self.rpc_command.pop()
-                    self.happy = True
+            ref = ref + (self.pos_offset)
+            # ref = ref + self.compute_offset(ref)
+            print('ref ' + str(ref))
+            print('pos ',str(pos))
+
+            # if ref < 0.8:
```

Comment:
we didn't go with the adaptive gains? Should we remove the commented lines, or leave them commented